test/test_exif2timestream.py

```diff
@@ -312,6 +322,37 @@ def test_generate_config_csv(self):
         e2t.generate_config_csv(out_csv)
         self._md5test(out_csv, "bf1ff915a42390a15ab8e4704e5c38e9")
 
+    # Tests for checking parsing of dates from filename
+    def test_check_date_parse(self):
+        self.assertEqual(e2t.get_time_from_filename("whroo20141101_001212M.jpg", "%Y%m%d_%H%M%S"), strptime("20141101_001212", "%Y%m%d_%H%M%S"))
+        self.assertEqual(e2t.get_time_from_filename("TRN-NC-DSC-01~640_2013_06_01_10_45_00_00.jpg", "%Y_%m_%d_%H_%M_%S"), strptime("2013_06_01_10_45_00", "%Y_%m_%d_%H_%M_%S"))
+
+    def test_check_write_exif(self):
+        # Write To Exif
+        filename = 'jpg/whroo20131104_020255M.jpg'
+        date_time = e2t.get_time_from_filename(path.join(self.camupload_dir, filename), "%Y%m%d_%H%M%S")
+        e2t.write_exif_date(path.join(self.camupload_dir, filename), date_time)
+
+        # Read From Exif
+        exif_tags = pexif.JpegFile.fromFile(path.join(self.camupload_dir, filename))
```

Comment:
I'd hard-code in your expected result, so that if `pexif` breaks you test, you know about it